app/javascript/dashboard/components/widgets/conversation/bubble/integrations/Dyte.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div>
+    <woot-button
+      size="small"
+      variant="smooth"
+      color-scheme="secondary"
+      icon="video-add"
+      class="join-call-button"
+      :is-loading="isLoading"
+      @click="joinTheCall"
+    >
+      {{ $t('INTEGRATION_SETTINGS.DYTE.CLICK_HERE_TO_JOIN') }}
+    </woot-button>
+    <div v-if="dyteAuthToken" class="video-call--container">
+      <iframe
+        :src="meetingLink"
+        allow="camera;microphone;fullscreen;display-capture;picture-in-picture;clipboard-write;"
+      />
```

Comment:
The integration is loaded via the iFrame option they provide so that we are not embedding the SDKs unnecessarily. This option needs to be revisited.